src/reader/DifficultyFeedbackBox.js

```diff
@@ -52,19 +56,19 @@ export default function DifficultyFeedbackBox({ api, articleID }) {
           const emojiComponent = () => {
             switch (option) {
               case 'Easy':
-                return isHovered === option ? (
+                return shouldBeMarked(option) ? (
                   <SentimentVerySatisfiedTwoToneIcon sx={emojiSize} />
```

Comment:
would this not be clear if it were refactored as `<SentimentVerySatisfiedIcon marked={shouldBeMarked(option)}>`?

Reply:
As we understand it, the SentimentVerySatisfiedTwoToneIcon is simply imported from MaterialUI, where there is a specific component for a *marked* one, and one for an *unmarked* one. If we were to add the *marked* parameter, we would have to make a custom component. We do not think that this would make sense to do for all marked-unmarked pairs of icons.

A possible solution would be to make a generic *MarkableIcon* that takes two icons and the *marked* parameter, but we are unsure whether this would even make the code more clear 🤔